### PR TITLE
Format only, global ArduPilot vs Ardupilot

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: dev/source/docs/sitl-simulator-software-in-the-loop.rst [hunk 1]
```diff
@@ -58,8 +58,8 @@ vary.  For instance the ports between ArduPilot and the simulator
 on the image are 5501/5502 but they can vary to be 5504/5505 or other
 port numbers depending on your environment.
 
-.. image:: ../images/ArdupilotSoftwareintheLoopSITL.jpg
-    :target: ../_images/ArdupilotSoftwareintheLoopSITL.jpg
+.. image:: ../images/ArduPilotSoftwareintheLoopSITL.jpg
```
Comment (rmackay9) [inline dev/source/docs/sitl-simulator-software-in-the-loop.rst]: This link to an image should probably be left unchanged..

**Comment by rmackay9** [comment]
This LGTM with the possible exception of the link to an image mentioned above

File: dev/source/docs/sitl-simulator-software-in-the-loop.rst (hunk 1 above)
Comment (Hwurzburg) [reply dev/source/docs/sitl-simulator-software-in-the-loop.rst]: yep..that was the last dir i did and forgot to run the complete wiki after...my daily cron job that pulls the master and builds the complete wiki, for me to check in the AM, would have caught it...but after the fact...
fixed

**Comment by rmackay9** [comment]
a wonderful change ..

**Pull request merged**